src/demo/DemoTestLoanService.java

```diff
@@ -897,8 +912,14 @@ public static void testUpdateLoanAccountGuarantees() throws MambuApiException {
 				+ result2.getGuarantees().size());
 	}
 
-	// / Transactions testing
-	public static void testDisburseLoanAccount() throws MambuApiException {
+	/**
+	 *  Test disburse loan account
+	 *
+	 * @return the dibusrsement transaction
+	 *
+	 * @throws MambuApiException in case the disbursement failed
+	 */
+	public static LoanTransaction testDisburseLoanAccount() throws MambuApiException {
 
 		System.out.println(methodName = "\nIn testDisburseLoanAccount");
```

Comment:
Not necessarily an issue (excluding consistency), but some methods use getEnclosingMethod().getName() while this one still has its name hardcoded.

Reply:
Changed for consistency